packages/amplify_api/android/src/main/kotlin/com/amazonaws/amplify/amplify_api/rest_api/FlutterRestApi.kt

```diff
@@ -28,239 +28,279 @@ typealias FunctionWithApiName = KFunction4<
         @ParameterName(name = "exceptionConsumer") Consumer<ApiException>,
         RestOperation?>
 
-class FlutterRestApi {
-
-    companion object {
-        private val LOG = Amplify.Logging.forNamespace("amplify:flutter:api")
-        private val handler = Handler(Looper.getMainLooper())
-        const val GET = "get"
-        const val POST = "post"
-        const val PUT = "put"
-        const val DELETE = "delete"
-        const val HEAD = "head"
-        const val PATCH = "patch"
-
-        private fun restFunctionHelper(
-                methodName: String,
-                flutterResult: Result,
-                request: Map<String, Any>,
-                functionWithoutApiName: FunctionWithoutApiName,
-                functionWithApiName: FunctionWithApiName
-        ) {
-            var cancelToken: String
-            var apiName: String?
-            var options: RestOptions
-
-            try {
-                cancelToken = FlutterApiRequest.getCancelToken(request)
-                apiName = FlutterApiRequest.getApiPath(request)
-                options = FlutterApiRequest.getRestOptions(request)
-
-                // Needed to prevent Android library from throwing a fatal error when body not present in some methods. https://github.com/aws-amplify/amplify-android/issues/1355
-                FlutterApiRequest.checkForEmptyBodyIfRequired(options, methodName)
-            } catch (e: ApiException) {
-                handler.post {
-                    ExceptionUtil.postExceptionToFlutterChannel(flutterResult, "ApiException",
-                            ExceptionUtil.createSerializedError(e))
-                }
-                return
-            } catch (e: Exception) {
-                handler.post {
-                    ExceptionUtil.postExceptionToFlutterChannel(flutterResult, "ApiException",
-                            ExceptionUtil.createSerializedUnrecognizedError(e))
-                }
-                return
-            }
+enum class RestOperationType(val value: String) {
```

Comment:
Same question - is this general refactoring?

Reply:
This was to address a lint warning triggered by the constants beforehand.